browser/ui/webui/playlist_ui.cc

```diff
@@ -81,7 +81,8 @@ class UntrustedPlayerUI : public ui::UntrustedWebUIController {
                     "chrome-untrusted://brave-resources 'unsafe-inline';"));
     source->OverrideContentSecurityPolicy(
         network::mojom::CSPDirectiveName::ImgSrc,
-        std::string("img-src 'self' chrome-untrusted://resources;"));
+        std::string("img-src 'self' chrome-untrusted://playlist-data "
+                    "chrome-untrusted://resources;"));
```

Comment:
I believe adding playlist-data to img-src CSP is okay as we already allowed this URL to media-src CSP.